HLTriggerOffline/Higgs/python/hltHiggsValidator_cfi.py

```diff
@@ -265,7 +265,7 @@
             "HLT_L1_TripleJet_VBF_v"
             ),
         recJetLabel  = cms.string("ak4PFJetsCHS"),
-        jetTagLabel  = cms.string("pfCombinedSecondaryVertexBJetTags"),
```

Comment:
this doesn't match the description of the PR "Remove legacy b-taggers"
Are this file changes appropriate in this PR?